rust/cubestore/cubestore/src/metastore/mod.rs

```diff
@@ -524,15 +544,50 @@ pub struct Index {
     #[serde(default)]
     partition_split_key_size: Option<u64>,
     #[serde(default)]
-    multi_index_id: Option<u64>
+    multi_index_id: Option<u64>,
+    #[serde(default = "Index::index_type_default")]
+    index_type: IndexType
 }
 }
 
+#[derive(Clone, Serialize, Deserialize, Debug, Eq, PartialEq, Hash)]
+pub enum AggregateFunction {
+    SUM = 1,
```

Comment:
No, we can't. Because they are different entities. For example, an HLL function  will be added to `AggregateFunction`. 
But it's not even that that matters, but the fact that these enums have different purposes. One defines the function in physical plans, the second defines the type of column in the table. Combining them into one enum can lead to unpleasant things when we need to work with such a single enum in different contexts